src/main/resources/db/migration/V028__UpdateNewCourtsNRO.sql

```diff
@@ -0,0 +1,275 @@
+    'Great Grimsby Combined Court',
+    'Inner London Crown Court',
+    'Kingston-upon-Hull Combined Court',
+    'Maidstone Combined Court',
+    'Manchester Crown Court',
+    'Merthyr Tydfil Combined Court',
+    'Mold Justice Centre',
+    'Newcastle upon Tyne Crown Court',
+    'Norwich Combined Court',
+    'Nottingham Crown Court',
+    'Oxford Combined Court',
+    'Peterborough Combined Court',
+    'Portsmouth Combined Court',
+    'Preston Combined Court',
+    'Sheffield Combined Court',
+    'Southampton Combined Court',
+    'Stafford Combined Court',
+    'Stoke-on-Trent Combined Court',
+    'Teesside Combined Court',
+    'Warwick Combined Court',
+    'Winchester Combined Court',
+    'Wolverhampton Combined Court',
+    'Leeds Combined Court',
+    'Leeds Magistrates'' Court',
+    'Liverpool Crown Court',
+    'Isle of Wight Combined Court',
+    'Birmingham Crown Court',
+    'Manchester Minshull Street Crown Court'
+);
+
+--Audit entry regarding location code updates
+INSERT INTO public.audits (
+    id, 
+    table_name, 
+    table_record_id, 
+    source, 
+    category, 
+    activity, 
+    functional_area, 
+    created_by, 
+    created_at, 
+    audit_details
+)
+SELECT
+    gen_random_uuid() AS id,
+    'courts' AS table_name,
+    c.id AS table_record_id,
+    'AUTO' AS source,
+    'Court' AS category,
+    'Update' AS activity,
+    'Update a court' AS functional_area,
+    (SELECT id FROM public.users WHERE last_name = 'Admin' LIMIT 1) AS created_by,
+    NOW() AS created_at,
+    jsonb_build_object(
+        'description', 'Court location code updated',
+        'courtName', c.name,
+        'locationCode', c.location_code
+    ) AS audit_details
+FROM public.courts c
+WHERE c.name IN (
+    'Newcastle upon Tyne Crown Court',
+    'Kingston-upon-Thames Crown Court',
+    'Leeds Magistrates'' Court'
+);
+
+-- Audit entry regarding deleted courts
+INSERT INTO public.audits (
+    id, 
+    table_name, 
+    table_record_id, 
+    source, 
+    category, 
+    activity, 
+    functional_area, 
+    created_by, 
+    created_at, 
+    audit_details
+)
+SELECT
+    gen_random_uuid() AS id,
+    'courts' AS table_name,
+    NULL AS table_record_id,
+    'AUTO' AS source,
+    'Court' AS category,
+    'Delete' AS activity,
+    'Delete a court' AS functional_area,
+    (SELECT id FROM public.users WHERE last_name = 'Admin' LIMIT 1) AS created_by,
+    NOW() AS created_at,
+    jsonb_build_object(
+        'description', 'Court has been deleted.',
+        'courtName', name
+    ) AS audit_details
+FROM (
+    VALUES
+        ('Caernarfon Justice Centre'),
+        ('Coventry Combined Court Centre'),
+        ('Doncaster Crown Court (Doncaster Justice Centre South)'),
+        ('King''s Lynn Crown Court'),
+        ('Lancaster Crown Court'),
+        ('Newcastle Moot Hall Annex'),
+        ('Newport (South Wales) Crown Court'),
+        ('Warrington Crown Court')
+) AS deleted_courts(name);
```

Comment:
Should probably add the court's id (similar to update audits and how the delete audits normally look). 
```suggestion
    c.id AS table_record_id,
    'AUTO' AS source,
    'Court' AS category,
    'Delete' AS activity,
    'Delete a court' AS functional_area,
    (SELECT id FROM public.users WHERE last_name = 'Admin' LIMIT 1) AS created_by,
    NOW() AS created_at,
    jsonb_build_object(
        'description', 'Court has been deleted.',
        'courtName', name
    ) AS audit_details
FROM (
    SELECT * FROM courts
    WHERE name in (
        'Caernarfon Justice Centre',
        'Coventry Combined Court Centre',
        'Doncaster Crown Court (Doncaster Justice Centre South)',
        'King''s Lynn Crown Court',
        'Lancaster Crown Court',
        'Newcastle Moot Hall Annex',
        'Newport (South Wales) Crown Court',
        'Warrington Crown Court'
) AS c;
```